WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormActionsFactory.swift

```diff
@@ -24,6 +24,9 @@ enum ProductFormEditAction {
     case variationName
     case noPriceWarning
     case status
+    // Non-core products only
```

Comment:
For making clear what are the non-core products types.
```suggestion
    // Non-core products only, es. products with subscriptions
```

Reply:
good idea! added in https://github.com/woocommerce/woocommerce-ios/pull/2694/commits/611a385d629495efbec9bf7752c1ef84259831f8